roxie/ccd/ccdmain.cpp

```diff
@@ -717,6 +717,20 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
         if (udpSnifferEnabled && !roxieMulticastEnabled)
             DBGLOG("WARNING: ignoring udpSnifferEnabled setting as multicast not enabled");
 
+        int ttlTmp = topology->getPropInt("@multicastTTL", 1);
```

Comment:
I would code it exactly as the other hundred-or-so other roxie settings are done... search the sources for preabortKeyedJoinsThreshold, for example. You will see the default value (100) appears in quite a few places, and you could argue that's less than idea, but consistency is also a good thing, and I don't expect we'll be changing these defaults frequently.

I'm not sure that you really need the ttlTmp variable TBH.

You should also probably update environment.xml.in